app/controllers/result_filters/provider_controller.rb

```diff
@@ -11,22 +12,27 @@ def new
       @provider_suggestions = Provider
         .select(:provider_code, :provider_name)
         .where(recruitment_cycle_year: Settings.current_cycle)
-        .with_params(search: params[:query])
+        .with_params(search: sanitized_params["query"])
```

Comment:
A user could potentially go straight to the /results/filter/provider controller direct (without being redirected there). Strip out non-ASCII characters again?

Reply:
Maybe on the rack middleware level.... this selective satanizing looks hellish